### DISCUSSION TITLE
Is it possible to interact with opened website in the browser?

### DISCUSSION THREAD

**Discussion by MarcinusX** [leancodepl/patrol#1050] [Q&A]
I have a flow in the app where I open a website inside a browser, do a few clicks there, and then the website redirects me back to my app. Is it possible to interact with browser through Patrol?

**Comment by bartekpacia** [comment]
Yup!

**Reply by lewandowski-jan** [reply]
https://patrol.leancode.co/native/usage#basics

**Reply by tjuros01** [reply]
Is it possible to scroll inside mobile browser (for example Safari)? I have case in which button is not visible until I scroll to it. @bartekpacia
